.packcheck.ignore

```diff
@@ -7,5 +17,7 @@ cabal.project.hpc-coveralls
 cabal.project.O0
 cabal.project.Werror
 cabal.project.Werror-nocode
+default.nix
+examples/README.md
```

Comment:
Shouldn't we package this?
Can we just remove it an add a note in the `README`?